src/soundanalysis.cs

```diff
@@ -33,6 +34,14 @@ enum SNErrorCode : long {
 		InvalidFile,
 	}
 
+	[Watch (8,0), TV (15,0), Mac (12,0), iOS (15,0)]
```

Comment:
is this also new in MacCatalyst ?
if not -> `[NoMacCalayst]`
if so -> [MacCatalyst (15,0)]`

Reply:
Good call, this should support MacCatalyst!